src/main/java/org/jenkins/tools/test/maven/ExternalMavenRunner.java

```diff
@@ -119,7 +121,7 @@ public void run() {
             try (InputStream is = p.getInputStream();
                     Reader isr = new InputStreamReader(is, Charset.defaultCharset());
                     BufferedReader r = new BufferedReader(isr);
-                    OutputStream os = new FileOutputStream(buildLogFile, true);
+                    OutputStream os = buildLogFile == null ? NullOutputStream.NULL_OUTPUT_STREAM : new FileOutputStream(buildLogFile, true);
```

Comment:
Why not use Java 11's [`OutputStream#nullOutputStream`](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/io/OutputStream.html#nullOutputStream())? Seems more maintainable than a third-party library.

Reply:
271e7a21fb99bc54d249a0ce5c417e6a924b33c9